firmware/src/core/Feeds.h

```diff
@@ -0,0 +1,25 @@
+#ifndef FEEDS_H
+#define FEEDS_H
+
+#include <Arduino.h>
+#include "Feed.h"
+#include "..\..\feeds\FeedFactory.h"
```

Comment:
You need to use forward slashes, linux chokes on the backslash, windows does not care so both work there.